### ISSUE TITLE
Ureal Plugins GetARTexture() didn't work

### ISSUE THREAD

**Issue by zhmxiaowo** [google-ar/arcore-unreal-sdk#85]
i wanna get the raw data from arcore camera and input to my location system ,I try to read the ARTexture data,but it can not be read as a ptr,what show i do

**Comment by Saikarthi** [comment]
did you find anything.? same prob here

**Comment by zhmxiaowo** [comment]
yes i did it,look my sdk repository:[SpaceTarget](https://github.com/zhmxiaowo/SpaceTarget)

there is a ar c++ subsystem,you can get the yuv data and translate it became to a texture2d
